platformio.ini

```diff
@@ -83,22 +91,30 @@ framework = arduino
 monitor_port = COM9
 build_flags=
 
+[env:promini@debug]
+platform = atmelavr
+board = pro8MHzatmega328
```

Comment:
Bist Du dir mit der 8 MHz Variante sicher?

Reply:
Eigentlich müssten hiervon zwei Varianten rein:

> This board was developed for applications and installations where space is premium and projects are made as permanent set ups. Small, available in 3.3 V and 5 V versions, powered by ATmega328P.

Der 3,3V wird mit 8 MHz getaktet und der 5V mit 16 MHz.

Reply:
*lach*

Diese stand vorher auch schon drin, ich habe diese nur verschoben in der Position. Du hattest diese damals so hereingenommen und möchtest nun "mir dies ankreiden"  *smile*

Reply:
Wenn das vorher schon enthalten war ist ja gut.
Läuft der miniCul womöglich auf 8Mhz.. war mir nicht mehr so bewusst gerade.

Reply:
Kein Problem :-)

Soll ich noch einen PR nachschieben für https://github.com/RFD-FHEM/SIGNALDuino/pull/128#discussion_r437678188 um das in einem Rutsch zu haben oder gibt es noch andere Dinge welche geändert werden sollen / müssten ?